docs/src/modules/components/ApiPage/table/PropertiesTable.tsx

```diff
@@ -41,6 +43,10 @@ const StyledTable = styled('table')(
       borderColor: alpha(darkTheme.palette.primary[100], 0.8),
       backgroundColor: `var(--muidocs-palette-primary-50, ${lightTheme.palette.primary[50]})`,
     },
+    '& .MuiApi-table-item-signature-type': {
+      textDecoration: 'underline',
```

Comment:
Nice DX improvement. Perhaps using the link styles, but with a dotted underline instead of solid would look nice for these:

<img width="303" height="117" alt="Screenshot 2025-07-16 at 15 32 32" src="https://github.com/user-attachments/assets/dfaf53f8-58c0-478a-a0b1-4c09f2deee55" />

```suggestion
      textDecoration: 'underline',
      textDecoration: 'dotted',
      textDecorationColor: alpha(lightTheme.palette.primary.main, 0.4),
      fontWeight: theme.typography.fontWeightMedium,
      color: `var(--muidocs-palette-primary-600, ${lightTheme.palette.primary[600]})`,
      '&:hover': {
        textDecorationColor: 'inherit',
      },
```

Also, what do you think about combining the param and type into a single `<code>`? Like this:

<img width="302" height="116" alt="Screenshot 2025-07-16 at 15 31 59" src="https://github.com/user-attachments/assets/e28e2b42-36dc-48b0-9cee-41ba19111554" />

Looks a bit tidier, IMO.

Reply:
Thanks for the suggestions, they look great! 💙 
I've tried addressing the dark theme myself.
WDYT? 🤔 

https://github.com/user-attachments/assets/cdcf8511-35f8-4efe-a0f3-bb394c926fa3


https://github.com/user-attachments/assets/b03bbf97-cb52-4533-8584-e7c3efbd1e50